src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -146,7 +146,7 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new CachePoolPrunerPass(), PassConfig::TYPE_AFTER_REMOVING);
         $this->addCompilerPassIfExists($container, FormPass::class);
         $container->addCompilerPass(new WorkflowGuardListenerPass());
-        $container->addCompilerPass(new ResettableServicePass());
+        $container->addCompilerPass(new ResettableServicePass(), PassConfig::TYPE_BEFORE_OPTIMIZATION, -32);
```

Comment:
It would be good to also update `ResettableServicePassTest` with the same type and priority